go.mod

```diff
@@ -3,30 +3,46 @@ module github.com/presslabs/mysql-operator
 go 1.13
 
 require (
-	github.com/blang/semver v3.5.0+incompatible
+	github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6 // indirect
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/coreos/go-etcd v2.0.0+incompatible // indirect
+	github.com/cpuguy83/go-md2man v1.0.10 // indirect
+	github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0 // indirect
+	github.com/fatih/color v1.9.0 // indirect
 	github.com/go-ini/ini v1.57.0
-	github.com/go-logr/logr v0.1.0
+	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/zapr v0.4.0
 	github.com/go-sql-driver/mysql v1.5.0
-	github.com/go-test/deep v1.0.2
+	github.com/go-test/deep v1.0.7
+	github.com/gobuffalo/flect v0.2.2 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/imdario/mergo v0.3.8
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.1
-
-	github.com/presslabs/controller-util v0.2.3
+	github.com/gophercloud/gophercloud v0.1.0 // indirect
+	github.com/imdario/mergo v0.3.11
```

Comment:
Don't update `mergo`

Reply:
I've updated it to `0.3.12` where the major bugs are fixed.